geth/account/importer_test.go

```diff
@@ -0,0 +1,80 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"fmt"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/golang/mock/gomock"
+	"github.com/status-im/status-go/extkeys"
+	"github.com/stretchr/testify/require"
+	"testing"
```

Comment:
Using `goimports` for formatting? Would put `testing` behind `fmt` and an empty line as separator to 3rd party packages. We discussed about it. I suggest adding it as format-on-write to your editor.

Reply:
We've refused goimport. Only gofmt.

Reply:
Just went through the discussion. IMHO no clear decision. Only linter has been changed and @tiabc now also switches to `goimports`. I'm with @adambabik and prefer `goimports` style. But it's okay this way, yes.